flags/eigendaflags/deprecated.go

```diff
@@ -0,0 +1,157 @@
+package eigendaflags
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/urfave/cli/v2"
+)
+
+// All of these flags are deprecated and will be removed in release v2.0.0
+// we leave them here with actions that crash the program to ensure they are not used,
+// and to make it easier for users to find the new flags (instead of silently crashing late during execution
+// because some flag's env var was changed but the user forgot to update it)
+var (
+	DeprecatedDisperserRPCFlagName                 = withDeprecatedFlagPrefix("disperser-rpc")
+	DeprecatedStatusQueryRetryIntervalFlagName     = withDeprecatedFlagPrefix("status-query-retry-interval")
+	DeprecatedStatusQueryTimeoutFlagName           = withDeprecatedFlagPrefix("status-query-timeout")
+	DeprecatedDisableTLSFlagName                   = withDeprecatedFlagPrefix("disable-tls")
+	DeprecatedResponseTimeoutFlagName              = withDeprecatedFlagPrefix("response-timeout")
+	DeprecatedCustomQuorumIDsFlagName              = withDeprecatedFlagPrefix("custom-quorum-ids")
+	DeprecatedSignerPrivateKeyHexFlagName          = withDeprecatedFlagPrefix("signer-private-key-hex")
+	DeprecatedPutBlobEncodingVersionFlagName       = withDeprecatedFlagPrefix("put-blob-encoding-version")
+	DeprecatedDisablePointVerificationModeFlagName = withDeprecatedFlagPrefix("disable-point-verification-mode")
+	DeprecatedWaitForFinalizationFlagName          = withDeprecatedFlagPrefix("wait-for-finalization")
+)
+
+func withDeprecatedFlagPrefix(s string) string {
+	return "eigenda-" + s
+}
+
+func withDeprecatedEnvPrefix(envPrefix, s string) []string {
+	return []string{envPrefix + "_" + s}
```

Comment:
for many ENV, there is not list of string, can you change

Reply:
For example, SERVICE_MANAGER_ADDR, list notation is confusing
<img width="1704" alt="Screenshot 2024-10-07 at 1 29 44 PM" src="https://github.com/user-attachments/assets/0754b519-7560-41c0-8daa-d4f9e2f10fc1">

The MEMstore one is better

<img width="1268" alt="Screenshot 2024-10-07 at 1 32 17 PM" src="https://github.com/user-attachments/assets/cbbf2e32-a4ca-40b0-a06c-fed74dd9d61b">

Reply:
[24e3aa0](https://github.com/Layr-Labs/eigenda-proxy/pull/165/commits/24e3aa0b0dbec3b77899c7e6f0e1152d517a91b6)

Reply:
Can you also change here, https://github.com/Layr-Labs/eigenda-proxy/blob/main/verify/cli.go#L44. 

With the latest change, it looks like this
<img width="1715" alt="Screenshot 2024-10-07 at 2 09 15 PM" src="https://github.com/user-attachments/assets/3bdc9989-dcce-4e8c-b8a3-1fc697d2e8d4">
Not sure why it has to be a list

Reply:
[e6f4439](https://github.com/Layr-Labs/eigenda-proxy/pull/165/commits/e6f44390510def223fadd9d6c9a002f64dfcc0cf)